ambry-commons/src/test/java/com.github.ambry.commons/BlobIdTest.java

```diff
@@ -603,6 +566,7 @@ private void assertBlobIdFieldValues(short version, BlobId blobId, BlobIdType ty
       default:
         fail("Unrecognized version");
     }
+    assertNotNull("getLongForm() should be supported for this version", blobId.getLongForm());
```

Comment:
minor: `".. for version " + blobId.getVersion()`?

Reply:
fixed